src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php

```diff
@@ -76,6 +83,10 @@ public static function register($mode = 0)
             $trace = debug_backtrace(true);
             $group = 'other';
 
+            $isVendor = (strpos($file, '/vendor/') !== false);
```

Comment:
@nicolas-grekas : we're outside the while on `$trace` here, so really, we're not looking at every file of the stack trace, but only at the file that has the error triggered in it.